buildSrc/build.gradle.kts

```diff
@@ -1,7 +1,21 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
 repositories {
     jcenter()
 }
 
 plugins {
     `kotlin-dsl`
+    kotlin("jvm") version "1.3.41"
```

Comment:
Can you explain why we need all of these kotlin changes in this PR?  Are they related?

Reply:
No. I just updated it as it's on my system.

Reply:
Ok let's revert these changes since they are not related to the feature.